opensrp-reveal/src/main/java/org/smartregister/reveal/interactor/BaseInteractor.java

```diff
@@ -511,4 +515,16 @@ public void findLastEvent(String eventBaseEntityId, String eventType) {
         });
 
     }
+    private String getLocationId(String locationName){
```

Comment:
Could we use the existing `locationRepository.getLocationByName()` method to fetch the locaiton?

Reply:
cool,will do.